databricks/koalas/base.py

```diff
@@ -37,6 +37,7 @@
     LongType,
     StringType,
     TimestampType,
+    NumericType,
```

Comment:
nit: shall we keep alphabetical order?

Reply:
Makes sense! I've used `IntegralType` instead and kept the alphabetical order.